data/init.js

```diff
@@ -17,15 +17,15 @@ const createTables = () => {
   const tableQueries = [
     `CREATE TABLE IF NOT EXISTS names (
       name_id INT PRIMARY KEY,
-      name TEXT
+      name_name TEXT
```

Comment:
Personal preference but I would think you only need the columns to be called id & name. The table name implies that they're related to product names/colors/sizes.

Reply:
The problem with having multiple columns named 'name' across tables is that node-postgres will only give you one of those columns when you query the db. There might be another way to remedy this but as it stands this was the most intuitive solution I could come up with.